lineman/app/services/mention_service.coffee

```diff
@@ -0,0 +1,7 @@
+angular.module('loomioApp').factory 'MentionService', ->
```

Comment:
I think this is a nice solution for the display issues... whenever we're looking to use a body of text for display, we call `prepareForDisplay` on it, whenever we want to edit that text, we call `prepareForEdit`